tests/rptest/archival/abs_client.py

```diff
@@ -94,6 +93,23 @@ def chunks(generator):
 
         return []
 
+    def move_object(self,
+                    bucket: str,
+                    src: str,
+                    dst: str,
+                    validate: bool = False,
+                    validation_timeout_sec=30):
+        # `validate` and `validation_timeout_sec` are unused, only present
+        # for compatibility with S3Client
+        container_client = ContainerClient.from_connection_string(
+            self.conn_str, container_name=bucket)
+
+        # There is no server-side "move" in ABS: this is a crude implementation,
+        # but good enough for tiny objects
+        content = container_client.download_blob(src).readall()
+        container_client.upload_blob(dst, content)
+        container_client.delete_blob(src)
```

Comment:
Should we return the status from here to check if the move worked? Or maybe check the result of these intermediate operations + log errors if any?

Reply:
As far as I can tell from the azure docs, these methods throw on error rather than returning response objects